src/ResultView.tsx

```diff
@@ -1,26 +1,145 @@
+          }}
+        />
+      </li>
+      <li>
+        <span className="sui-result__label">Role: </span>
+        <span
+          className="sui-result__value"
+          dangerouslySetInnerHTML={{
+            __html: role && role.snippet ? role.snippet : "NA",
+          }}
+        />
+      </li>
+      <li>
+        <span className="sui-result__label">Organisation: </span>
+        <span
+          className="sui-result__value"
+          dangerouslySetInnerHTML={{
+            __html:
+              organisation && organisation.snippet
+                ? organisation.snippet
+                : "NA",
+          }}
+        />
+      </li>
+      <li>
+        <span className="sui-result__label">School graduated from: </span>
+        <span
+          className="sui-result__value"
+          dangerouslySetInnerHTML={{
+            __html: school && school.snippet ? school.snippet : "NA",
+          }}
+        />
+      </li>
+      <li>
+        <span className="sui-result__label">Course of study: </span>
+        <span
+          className="sui-result__value"
+          dangerouslySetInnerHTML={{
+            __html:
+              course_of_study && course_of_study.snippet
+                ? course_of_study.snippet
+                : "NA",
+          }}
+        />
+      </li>
+      <li>
+        <span className="sui-result__label">Biography: </span>
+        <span
+          className="sui-result__value"
+          dangerouslySetInnerHTML={{
+            __html:
+              full_bio && full_bio.snippet ? full_bio.snippet + "..." : "NA",
+          }}
+        />
+      </li>
+    </ul>
+  );
+};
 
+const ResultTitleMobile = ({ title }: { title: any }) => {
+  const { name } = title;
+  return (
+    <span
+      className="sui-result__title"
+      dangerouslySetInnerHTML={{
+        __html: name && name.snippet ? name.snippet : "NA",
+      }}
+      style={{ textAlign: "center", width: "100%" }}
+    />
+  );
+};
+
+const ResultTitle = ({ title }: { title: any }) => {
+  const { name } = title;
+  return (
+    <span
+      className="sui-result__title"
+      dangerouslySetInnerHTML={{
+        __html: name && name.snippet ? name.snippet : "NA",
+      }}
+    />
+  );
+};
+
+const ResultView = ({ result }: { result: any }) => {
+  const MobileView = useMediaQuery("(max-width:800px)");
+  if (MobileView) {
+    return (
+      <div>
+        <li className="sui-result">
+          <div className="sui-result__header">
+            <ResultTitleMobile title={result} />
+          </div>
+          <div
+            className="sui-result__image"
+            style={{
+              width: "100%",
+              display: "flex",
+              justifyContent: "center",
+              paddingLeft: "0px",
+              paddingTop: "0px",
+            }}
+          >
+            <img
+              src={result.thumbnail_image_url.raw}
+              alt="thumb"
```

Comment:
Sorry, could you remove this line along with the `alt` text for the desktop version? I missed when merging a previous PR. This is not a useful alt-text and should just be removed.